diff --git a/server/config/env/production.js b/server/config/env/production.js
index f4d1b7c..8625cdc 100644
--- a/server/config/env/production.js
+++ b/server/config/env/production.js
@@ -306,7 +306,11 @@ module.exports = {
    *
    */

-  // ssl: undefined,
+  ssl: {
+    ca: require('fs').readFileSync(require('path').resolve(__dirname,'../ssl/ca.crt')),
+    key: require('fs').readFileSync(require('path').resolve(__dirname,'../ssl/private.key')),
+    cert: require('fs').readFileSync(require('path').resolve(__dirname,'../ssl/cert.crt'))
+  },

   /**
    *
diff --git a/server/config/security.js b/server/config/security.js
index acdb1ef..f481260 100644
--- a/server/config/security.js
+++ b/server/config/security.js
@@ -29,7 +29,7 @@ module.exports.security = {

   cors: {
     allRoutes: true,
-    allowOrigins: ['http://localhost:3000'],
+    allowOrigins: ['https://localhost:6000'],
     allowRequestHeaders: ['Authorization'],
     allowCredentials: false,
   },
